core_backend/app/routers/manage_content.py

```diff
@@ -87,9 +120,29 @@ async def retrieve_content(
     return contents
 
 
-@router.delete("/{content_id}/delete")
+@router.get("/summary", response_model=list[ContentSummary])
+async def retrieve_content_summary(
+    readonly_access_user: Annotated[
+        AuthenticatedUser, Depends(get_current_readonly_user)
+    ],
+    language: int,
```

Comment:
We should change the name to this!
```suggestion
    language_id: int,
```